ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Test the connectivity for VIP's
+  command: ping -c3 -i 3 {{ apivip }}
+  register: results
+  tags: validation
+
+- name: Print the ping results
+  debug:
+    var: results.stdout
+    verbosity: 2
+
+- name: If VIP's are being used fail with an error message
+  fail:
+    msg: "{{ apivip }} and {{ ingressvip }} are being used in somewhere else, kindly off the nodes to re-run the deployment"
```

Comment:
This message says its checking bot apivip and ingressvip but I only see a ping test for apivip above